docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -18,91 +18,86 @@ The tutorial aims to be an extension to the quickstart to help you better unders
 
 ## 1: Initialize Workflow Directory
 Bootstrap a new workflow directory by running `latch init` from the command line.
-```
-latch init covid-wf
+```shell-session
+latch init covid-wf -t subprocess
 ```
 Output:
-```
-Created a latch workflow called covid-wf.
+```shell-session
+Created a latch workflow in `covid-wf`
+Run
+        $ latch register covid-wf
+To register the workflow with console.latch.bio.
```

Comment:
```suggestion
```shell-session
$ latch init covid-wf -t subprocess
Created a latch workflow in `covid-wf`
Run
        $ latch register covid-wf
To register the workflow with console.latch.bio.
```
```